aspnetcore/blazor/components/render-modes.md

```diff
@@ -490,3 +512,49 @@ The following example includes the assembly of the `DifferentAssemblyCounter` co
 app.MapRazorComponents<App>()
     .AddAdditionalAssemblies(typeof(DifferentAssemblyCounter).Assembly);
 ```
+
+## Closure of circuits when there are no remaining interactive server components
+
+[!INCLUDE[](~/blazor/includes/closure-of-circuits.md)]
+
+## Custom shorthand render modes
+
+Whatever follows an `@rendermode` directive must be a valid Razor expression that would make sense as a method parameter in C#.
```

Comment:
I think this text can be simplified: The `@rendermode` directive takes a single parameter that is a static instance of type `IComponentRenderMode`. The `@rendermode` directive attribute can take any render mode instance (static or not). You can supply that render mode instance however you'd like. We provide the `RenderMode` static class with some predefined render modes for convenience, but you can of course create your own.

Reply:
Yes, I like your (short and clear) lead in to the example. I think I can resolve this, and we'll take a look on the next commit.